frontend/catalyst/qfunc.py

```diff
@@ -121,7 +157,7 @@ def _eval_quantum(*args):
         return tree_unflatten(out_tree_promise(), res_flat)[0]
 
 
-# pylint: disable=protected-access,no-member,not-callable
+# pylint: disable=not-callable
```

Comment:
This is wild, didn't vincent just add these? Truly a headache 😭

Reply:
I'll add them back and see if CodeFactor complains 😓

Reply:
The formatting check failed due to useless suppressions. See [here](https://github.com/PennyLaneAI/catalyst/actions/runs/9506271437/job/26203073536?pr=798).

Reply:
Formatting/linting should never fail due to the useless suppression warning, since it is just "informational" (IXXX). In that case it was and actual linting error that failed the check (E1101).